### PR TITLE
Update to core beta8

### PR DESCRIPTION
**Pull request by timbussmann**

### CODE REVIEW

**Comment by andreasohlund** [comment]
Failing tests

File: src/NServiceBus.Testing/NServiceBus.Testing.csproj [hunk 1]
```diff
@@ -7,8 +7,6 @@
     <OutputPath>..\..\binaries\</OutputPath>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
```
Comment (bording) [inline src/NServiceBus.Testing/NServiceBus.Testing.csproj]: This line isn't needed when using Particular.Packaging.

File: src/NServiceBus.Testing/NServiceBus.Testing.csproj [hunk 2]
```diff
@@ -17,10 +15,10 @@
   
   <ItemGroup>
     <PackageReference Include="Fody" Version="2.*" PrivateAssets="All" />
```
Comment (bording) [inline src/NServiceBus.Testing/NServiceBus.Testing.csproj]: Why not remove these wilcards as well?

Comment (timbussmann) [reply src/NServiceBus.Testing/NServiceBus.Testing.csproj]: missed that, will fix it

**Comment by timbussmann** [comment]
@bording should be fixed now.

**Pull request merged**